src/main/java/com/minecolonies/coremod/colony/requestsystem/data/StandardRequestResolverRequestAssignmentDataStore.java

```diff
@@ -0,0 +1,127 @@
+package com.minecolonies.coremod.colony.requestsystem.data;
+
+import com.google.common.reflect.TypeToken;
+import com.minecolonies.api.colony.requestsystem.StandardFactoryController;
+import com.minecolonies.api.colony.requestsystem.data.IProviderResolverAssignmentDataStore;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.colony.requestsystem.data.IProviderResolverAssignmentDataStore'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)